### DISCUSSION TITLE
What is a Micronaut Release?

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-platform#200] [General]
A Micronaut Framework 3.x release = Micronaut Starter Release + Micronaut Core Release
A Micronaut Framework 4.x release = Micronaut Platform Release + Micronaut Starter Release

Hence, this project version starts with [4.0.0](https://github.com/micronaut-projects/micronaut-platform/blob/master/gradle.properties#L1)

@micronaut-projects/core-developers any concerns?

**Comment by melix** [comment]
Sounds good to me!

**Comment by alvarosanchez** [comment]
The only caveat I see is that once Core is "yet another module", having it versioned with 4.x too can be confusing at the beginning. But perhaps there's no other way.

**Comment by Hc747** [comment]
Howdy! Does this project, `micronaut-platform`, have a seperate release cadence to other micronaut projects? I notice at the time of posting this that other projects are at version `4.1.0` and this project is at `4.0.3`. I use the `micronaut-platform` project BOM as a version catalog within Gradle, so I'm unable to update to the most recent version until this project is similarly updated. Cheers.

**Reply by yawkat** [reply]
For 4.x, we decoupled micronaut-platform (which could be called the "general" micronaut version) and micronaut-core. There is a new micronaut-core release, but no micronaut-platform release containing it yet. You can manually specify the version of the micronaut-core modules you use if you wish to upgrade.

**Reply by Hc747** [reply]
Would the recommended approach be to use the micronaut-core version catalog, or wait until a subsequent release of the micronaut-platform version catalog contains the updated version of MN-core?

**Reply by sdelamo** [reply]
@Hc747 Micronaut Platform will update to Micronaut Core 4.1.0 or later in Micronaut Platform 4.1.0. We typically release minor versions 6 to 12 weeks after the previous minor. We release 4.0.0 July 14th. Thus you can expect Micronaut Framework 4.1.0 last week of August or first weeks of september September.

**Comment by nedelva** [comment]
When can we expect a Micronaut Platform release that includes the micronaut-core version 4.3.x?
